frontend/src/components/layout/navigation/UpgradeButton.tsx

```diff
@@ -0,0 +1,25 @@
+import { useLocalization } from "@fluent/react";
+import styles from "./UpgradeButton.module.scss";
+import { useGaViewPing } from "../../../hooks/gaViewPing";
+import Link from "next/link";
+import { trackPurchaseStart } from "../../../functions/trackPurchase";
+
+export const UpgradeButton = (): JSX.Element => {
+  const { l10n } = useLocalization();
+  const upgradeButtonRef = useGaViewPing({
+    category: "Purchase Button",
+    label: "navbar-upgrade-button",
+  });
+
+  return (
+    <Link href="/premium">
+      <a
+        ref={upgradeButtonRef}
+        onClick={() => trackPurchaseStart({ label: "navbar-upgrade-button" })}
+        id={styles["upgrade-button"]}
```

Comment:
Just wondering: what was the reason for switching from `className` to `id`?

Reply:
There is styling that overrides the upgrade button. Specifically, when `theme` comes into play, Eduardo wanted the upgrade button specifically to have a hover different than the other menu items (color).

Reply:
Gotcha - I guess CSS modules don't "protect" against cascading styles when a module is contained within another module :/